tldw_Server_API/app/core/DB_Management/backends/factory.py

```
@@ -46,6 +54,240 @@ def _describe_sqlite_target(config: DatabaseConfig) -> str:
+        _normalize_sqlite_target(raw_path),
+        bool(config.sqlite_wal_mode),
+        bool(config.sqlite_foreign_keys),
+    )
+
+
+def _close_backend_instance(name: str, backend: DatabaseBackend) -> None:
+    try:
+        if hasattr(backend, "_pool") and backend._pool:
+            backend._pool.close_all()
+        logger.info(f"Closed backend: {name}")
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)